lib/simplecov/configuration.rb

```diff
@@ -196,6 +196,40 @@ def at_exit(&block)
       @at_exit ||= proc { SimpleCov.result.format! }
     end
 
+    #
+    # Gets or sets the behavior to start a new forked Process.
+    #
+    # By default, it will add " (Process #{pid})" to the command_name, and start SimpleCov in quiet mode
+    #
+    # Configure with:
+    #
+    #     SimpleCov.at_fork do |pid|
+    #       SimpleCov.start do
+    #         # This needs a unique name so it won't be ovewritten
+    #         SimpleCov.command_name "#{SimpleCov.command_name} (subprocess: #{pid})"
+    #         # be quiet, the parent process will be in charge of using the regular formatter and checking coverage totals
+    #         SimpleCov.print_error_status = false
+    #         SimpleCov.formatter = SimpleCov::Formatter::SimpleFormatter
+    #         SimpleCov.minimum_coverage 0
+    #         # start
+    #         SimpleCov.start
+    #       end
+    #     end
+    #
+    def at_fork(&block)
+      @at_fork = block if block_given?
+      @at_fork ||= lambda { |pid|
+        # This needs a unique name so it won't be ovewritten
+        SimpleCov.command_name "#{SimpleCov.command_name} (subprocess: #{pid})"
+        # be quiet, the parent process will be in charge of using the regular formatter and checking coverage totals
+        SimpleCov.print_error_status = false
+        SimpleCov.formatter = SimpleCov::Formatter::SimpleFormatter
```

Comment:
do we need to set the formatter here?

Reply:
My theory was making the computer do less work, but it's maybe not necessary.

Reply:
turns out at least the SimpleCov::Formatter::HTMLFromatter spams stdout

Reply:
yeah it looks like as it is currently setting SimpleCov.formatter to nil raises a nil.new NoMethodError